ejerciciosProg16-5/suma2Array.ts

```diff
@@ -40,10 +40,9 @@ botonaso.addEventListener("click", () => {
 
     for (let elemento: number = 0; elemento < nElementos; elemento++) {
       arrayResultado[elemento] = array1[elemento] + array2[elemento];
-    }
-
-    for (let elemento: number = 0; elemento < nElementos; elemento++) {
       console.log(arrayResultado[elemento]);
     }

```

Comment:
Borramos el **for** del console.log y agregamos el console.log en el mismo **for** donde se hace la suma.
Así nos ahorramos hacer otro for.